geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -273,9 +274,16 @@ class SelectionMapFragment(
                 }
                 map.setMarkerIcon(featureId, item.largeIcon)
                 summarySheet.setItem(item)
-                selectedFeatureViewModel.setSelectedFeatureId(featureId)
 
-                summarySheetBehavior.state = STATE_EXPANDED
+                summarySheetBehavior.state = STATE_COLLAPSED
+                summarySheet.viewTreeObserver.addOnGlobalLayoutListener(object : ViewTreeObserver.OnGlobalLayoutListener {
```

Comment:
We need to use a `GlobalLayoutListener` here as otherwise the `peekHeight` could end up being calculated before the view has rendered and be inaccurate.